content/blog/sharing_localwp_wordPress_sites.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Sharing LocalWP WordPress sites"
+description: "Easily share your Local WordPress sites online with Pinggy. This guide covers the steps to securely share LocalWP setups for collaboration and remote access with minimal setup."
+date: 2024-11-01T10:00:00+05:30
+draft: false
+tags: ["WordPress", "Pinggy", "LocalWP", "guide", "tunneling", "remote access"]
+og_image: "/blog_img/wordpress_pinggy/share_wordpress_online.webp"
+outputs:
+  - HTML
+  - AMP
+---
+
+{{< image "sharing_localwp_wordPress_sites/thumbnail.webp" "How to Easily Share ComfyUI Online" >}}
+
+Creating a {{< link href="https://localwp.com/" >}}local WordPress{{< /link >}} site is a routine for most developers, though sharing a live preview with clients or multiperson collaboration in real-time without having to waste time on complicated networks and open ports is always a big problem. That’s where Pinggy comes into play – a simple and lightweight tool for sharing local WordPress sites, that works in harmony with LocalWP by Flywheel tool, helps to tunnel your site in order not to trouble yourself with its remote accessibility. This tutorial will help you to find out how you can make the most of Pinggy if you have a local WordPress environment set up.
+
+{{% tldr %}}
+
+1. **Configure Pinggy Pro**:
+   - **Step 1**: Get a persistent subdomain from Pinggy, such as `mysite.a.pinggy.link`.
+   - **Step 2**: Set this subdomain as the site domain in LocalWP.
+2. **Run Command**: Open terminal and run:
+   ```bash
+   ssh -p 443 -L4300:localhost:4300 -o StrictHostKeyChecking=no -o ServerAliveInterval=30 -t -R0:your-subdomain.a.pinggy.link:80 <your-token>@a.pinggy.io
+   ```
+3. **Access Site**: Use the generated Pinggy URL to access your local WordPress site remotely.
+
+{{% /tldr %}}
+
+
+{{< iframe src="https://www.youtube.com/embed/DwMt-Ts-1Vk?si=8CRKOBp7xoCKaHnI" title="YouTube video player" >}}
+
+## Why Use Pinggy for Local WordPress Site Sharing?
+Pinggy allows you to connect to Local sites from outside your network without the need for a VPN, or even to change firewalls. It’s especially useful for developers working with **LocalWP by Flywheel**, what helps to solve usual issues like reverse proxying, secure connections or mixed content. Pinggy in one way or another provides its users with the means to remotely access local WordPress setups, and as such, is an essential tool for developers to create and test their sites.
+
+
+### Prerequisites
+
+To get started with [Pinggy](https://pinggy.io/) for Creating a {{< link href="https://localwp.com/" >}}local WordPress{{< /link >}} site sharing, you’ll need the following:
```

Comment:
Link to LocalWP, not local wordpress. Local wordpress is not same as LocalWP. LocalWP is a software. Locally wordpress can be run without LocalWP.